GoDaddy.sh

```diff
@@ -20,22 +20,22 @@
 # Begin settings
 # Get the Production API key/secret from https://developer.godaddy.com/keys/.
 # Ensure it's for "Production" as first time it's created for "Test".
-Key=fsvVddxVsfe_frhN1SAgfdgCdAfyVdWDdv
-Secret=FwkUfafh8dgrsdydrdfC1e
+KEY=${KEY}
+SECRET=${SECRET}
  
-# Domain to update.
-Domain=teanazar.com
+# DOMAIN to update.
+DOMAIN=${DOMAIN}
  
 # Advanced settings - change only if you know what you're doing :-)
 # Record type, as seen in the DNS setup page, default A.
-Type=A
+Type=${TYPE}
```

Comment:
KEY,SECRET, and DOMAIN use uppercase local variable names.
Type,Name is lowercase.
Any other reasons?

Reply:
How about following google's shell code convention?
https://google.github.io/styleguide/shellguide.html#s7-naming-conventions

Reply:
I can create a jira for fixing this and then work on it.